mailpoet/lib/Automation/Engine/Data/Automation.php

```diff
@@ -160,6 +160,30 @@ public function needsFullValidation(): bool {
     return in_array($this->status, [Automation::STATUS_ACTIVE, Automation::STATUS_DEACTIVATING], true);
   }
 
+  public function getPositionOfStep(Step $step): int {
+    return $this->calculatePositionOfStep($step);
+  }
+
+  private function calculatePositionOfStep(Step $step, string $currentStepId = 'root', int $position = 0): int {
```

Comment:
I do not see how it could have multiple positions. My idea was something like that:
```

	[POS 0: root]
		|
	       / \
	      /   \
	     /     \
            /       \
     [POS 1: T1]  [POS 1: T2]
           \        /
            \      /
             \    /
              \  /
               \/
               |
        [POS 2: Action]
```

Do you mean a second trigger could maybe first link to another action before coming to the send email step? :thinking: